packages/design-tokens/src/default/scss/_colors.scss

```diff
@@ -1,330 +1,189 @@
 $white: #fff !default;
-$gray-100: #f8f7f7 !default;
+$gray-100: #f4f4f4 !default;
 $gray-200: #ebeaea !default;
-$gray-300: #dbdbdb !default;
-$gray-400: #c4c4c4 !default;
-$gray-500: #90a2a7 !default;
-$gray-600: #5a6a6e !default;
-$gray-700: #404b4e !default;
-$gray-800: #343a40 !default;
-$gray-900: #091417 !default;
+$gray-300: #e9e9e9 !default;
+$gray-400: #dbdbdb !default;
+$gray-500: #d4d4d4 !default;
+$gray-600: #c4c4c4 !default;
+$gray-800: #737373 !default;
 $black: #000 !default;
 
-$info-200: #ebf3f8 !default;
-$info-400: #90cff1 !default;
-$info-600: #85c1e1 !default;
-$info-800: #78a9c5 !default;
+$glacier-300: #77a6c0 !default;
+$glacier-400: #678fa6 !default;
+$glacier-500: #57798c !default;
 
-$success-200: #f2f5e7 !default;
-$success-400: #b7d16d !default;
-$success-600: #9ebf40 !default;
-$success-800: #8fad3a !default;
+$porcelain-050: #f0f4f5 !default;
```

Comment:
Do you insist on leading zero? I would remove it.

Reply:
Nope, removed.